dm/pkg/utils/db.go

```diff
@@ -632,3 +632,31 @@ func GetTableCreateSQL(ctx context.Context, conn *sql.Conn, tableID string) (sql
 	}
 	return createStr, nil
 }
+
+// GetTableCollation gets table collation info by 'SHOW TABLE STATUS'.
+// See. https://dev.mysql.com/doc/refman/5.7/en/show-table-status.html
```

Comment:
```suggestion
```
maybe show the resp from db [like other func](https://github.com/pingcap/ticdc/blob/0076d097b2ab252ca81b7a05430c90f34b73754d/dm/pkg/utils/db.go#L193) is better than give a link here

Reply:
ok